server/src/schemas/storeFrontSchema.ts

```diff
@@ -0,0 +1,74 @@
+import { gql } from "apollo-server-express";
+
+export const StoreFrontTypeDefs = gql`
+
+	scalar Date
+
+	type InventoryItem {
+		id: ID!;
+		image: String;
+		name: String;
+		labels: [String];
```

Comment:
Should we have a `Label` type instead of just strings? Just asking - I could be convinced either way.

Reply:
What would be in a label type? just a string?

Reply:
I was thinking the `Label` could have an ID and a string name. That way, if an admin renames a label from the portal, we can just update the name property.

However, I think we could still manage this with the way you have it, and I think my way might be overcomplicating things, so let's just keep it as-is.